### PR TITLE
Deleteroute

### PR DESCRIPTION
**Pull request by SvLowman**

### CODE REVIEW

File: client/src/pages/TaggingPage.js [hunk 1]
```diff
@@ -53,22 +53,28 @@ const TaggingPage = ({ selectedImage, setSelectedImage }) => {
...
+  }, [tagArray, allImages, refetch]);
+
+  const handleImageDelete = async () => {
+    await deleteImageObj(userName, imgNr);
+    setAllImages([...allImages]);
+  };
+
+  const handleTagDelete = async (tag) => {
+    await deleteTagItem(userName, imgNr, tag);
+    setTagArray([...tagArray]);
```
Comment (phlgr) [inline client/src/pages/TaggingPage.js:76]: Why don't you receive an Array in the first place?

File: server.js [hunk 2]
```diff
@@ -73,6 +79,41 @@ app.post(
...
+// DELETE-Routes to MongoDB
+app.delete("/api/users/:userName/images/:imgNr", async (request, response) => {
+  const { imgNr } = request.params;
+  try {
+    const imgDel = await deleteImage(imgNr);
+    if (imgDel.deletedCount === 0) {
+      response.status(404).send("Not found");
+      return;
+    }
+    response.send(`Image ${imgNr} deleted.`);
```
Comment (phlgr) [inline server.js:91]:
```suggestion
    response.status(200).send(`Image ${imgNr} deleted.`);
```

File: server.js [hunk 3]
```diff
@@ -73,6 +79,41 @@ app.post(
...
+  async (request, response) => {
+    const { imgNr } = request.params;
+    const { tagName } = request.params;
+    try {
+      const tagDel = await deleteTag(imgNr, tagName);
+      if (tagDel.modifiedCount === 0) {
+        response.status(404).send("Not found");
+        return;
+      }
+      response.send(`Tag ${tagName} deleted; ${tagDel.modifiedCount}`);
```
Comment (phlgr) [inline server.js:109]:
```suggestion
      response.status(200).send(`Tag ${tagName} deleted; ${tagDel.modifiedCount}`);
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: server.js (hunk 3 above)
Comment (SvLowman) [reply server.js:109]: Thanks for reviewing!

**Pull request merged**